modules/nw-installation-ingress-config-asset.adoc

```diff
@@ -20,14 +19,8 @@ spec:
   domain: apps.openshiftdemos.com
 ----
 
-The installation program stores this asset in the
-`cluster-ingress-02-config.yml` file in
-the `manifests/` directory. This `Ingress` resource defines the cluster-wide
-configuration for Ingress. This Ingress configuration is used as follows:
+The installation program stores this asset in the `cluster-ingress-02-config.yml` file in the `manifests/` directory. This `Ingress` resource defines the cluster-wide configuration for Ingress. This Ingress configuration is used as follows:
 
-* The Ingress Operator uses the domain from the cluster Ingress configuration as
-the domain for the default Ingress Controller.
+* The Ingress Operator uses the domain from the cluster Ingress configuration as the domain for the default Ingress Controller.
 
-* The OpenShift API server operator uses the domain from the cluster Ingress
-configuration as the domain used when generating a default host for a `Route`
-resource that does not specify an explicit host.
+* The OpenShift API Server Operator uses the domain from the cluster Ingress configuration as the domain. This domain is also used when generating a default host for a `Route` resource that does not specify an explicit host.
```

Comment:
This is the only bullet with a content change. The rest of the changes are to remove hard returns.

Reply:
Old content: 
"The OpenShift API server operator uses the domain from the cluster Ingress configuration as the domain used when generating a default host for a `Route` resource that does not specify an explicit host."

New content: 
"The OpenShift API Server Operator uses the domain from the cluster Ingress configuration as the domain. This domain is also used when generating a default host for a `Route` resource that does not specify an explicit host."